scripts/recordid-to-filepath.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script returns the exact path of a record.
+# Pass the id as the first argument.
+# On finding the correct file, it prints its path to the console.
+
+if [ $# -eq 0 ]; then
+  echo "Error: No record ID provided."
+  exit 1
+fi
+
+result=$(
+  grep -ri "$1" /content/data/records/ |
+  grep  '"recid"' |
```

Comment:
this grep would catch recid references inside the document, like `pileup->links->recid`. It might be better to look for regular expresion (like `grep -e '^    "recid":'`

Reply:
I think then it is better to just include the command mentioned on the top in the dev documentation. :+1:

Mine worked with multiple random scripts I tested, that's why I thought it should work fine, but it does not on some others now I've seen.